src/org/starexec/backend/GridEngineBackend.java

```diff
@@ -251,17 +270,23 @@ public String getRunningJobsStatus() {
     		envp[1] = "SGE_ROOT="+BACKEND_ROOT; // it seems we need to set this explicitly if we change the environment.
     		String results = Util.executeCommand(QUEUE_STATS_COMMAND,envp);
 
+		log.info(results);
```

Comment:
Just wondering why you guys want to dump this to the log.

Reply:
I don't have any reason to log or not log any of these things. I think that Rahul had just inserted some logs to understand his changes better, but I'll just remove them.